app/src/androidTest/java/org/mozilla/focus/activity/BrowsingHistoryTest.java

```diff
@@ -154,38 +155,25 @@ public void browsingTwoWebSites_sitesAreDisplayedInOrderInHistoryPanel() {
         onView(withId(R.id.browsing_history_recycler_view))
                 .check(matches(atPosition(2, hasDescendant(withText(targetUrlSite1)))));
 
-    }
-
-    /**
-     * Test case no: his_002
-     * Test case name: Visit a page via history record
-     * Steps:
-     * 1. Follow steps in his_001
-     * 2. Tap on a history
-     * 3. Navigate to the selected history site */
-    @Test
-    public void browsingWebsiteThenTapTheFirstHistoryItem_browsingHistoryRecordCorrectly() {
-
-        final String targetUrl = browsingWebsiteAndOpenHistoryPanel();
-
-        // Click the first item in history panel, index 0 is date label so index 1 is the latest one of history item
+        // Click the first item in history panel, index 0 is date label so index 2 is the latest one of history item
         onView(ViewMatchers.withId(R.id.browsing_history_recycler_view))
-                .perform(RecyclerViewActions.actionOnItemAtPosition(1, click()));
+                .perform(RecyclerViewActions.actionOnItemAtPosition(2, click()));
```

Comment:
I'm first confused why it's 2 here. 
After reading the code, I found that In the test case, you clicked two websites, so you want to make sure the 2nd URL is shown in the 1st position, and the 1st URL is shown in the 2nd position.  
But this behaviour is not stated in the comment of this method.